### PR TITLE
Add plugins.portal.url.override parameter to promotion builds

### PR DESCRIPTION
**Pull request by blindpirate**
Because we saw network failures.

### CODE REVIEW

File: .teamcity/src/main/kotlin/promotion/BasePublishGradleDistribution.kt [hunk 1]
```diff
@@ -66,6 +66,6 @@ fun BuildSteps.buildStep(extraParameters: String, gitUserName: String, gitUserEm
     gradleWrapper {
         name = "Promote"
         tasks = "$prepTask $stepTask"
-        gradleParams = """-PcommitId=%dep.${RelativeId("Check_Stage_${triggerName}_Trigger")}.build.vcs.number% $extraParameters "-PgitUserName=$gitUserName" "-PgitUserEmail=$gitUserEmail" %additional.gradle.parameters% """
+        gradleParams = """-PcommitId=%dep.${RelativeId("Check_Stage_${triggerName}_Trigger")}.build.vcs.number% $extraParameters "-PgitUserName=$gitUserName" "-PgitUserEmail=$gitUserEmail" -Dorg.gradle.internal.plugins.portal.url.override=%gradle.plugins.portal.url% %additional.gradle.parameters%"""
```
Comment (wolfs) [inline .teamcity/src/main/kotlin/promotion/BasePublishGradleDistribution.kt:69]: Can we extract this parameter for overriding the plugin URL to some extension function/String constant?

Comment (blindpirate) [reply .teamcity/src/main/kotlin/promotion/BasePublishGradleDistribution.kt]: Done, PTAL!

**Review by wolfs — APPROVED** [review]
LGTM!

**Comment by blindpirate** [comment]
@bot-gradle test and merge

**Pull request merged**